package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "github-readme-stats",
-  "version": "1.0.0",
+  "name": "@zo-bro-23/github-readme-stats-test",
+  "version": "1.0.5",
   "description": "Dynamically generate stats for your GitHub readme",
```

Comment:
This is only temporary. I will change it to the actual package name before merging.